src/adaptation/icap/ModXact.cc

```diff
@@ -78,7 +78,7 @@ Adaptation::Icap::ModXact::ModXact(Http::Message *virginHeader,
     icapReply = new HttpReply;
     icapReply->protoPrefix = "ICAP/"; // TODO: make an IcapReply class?
 
-    debugs(93,7, HERE << "initialized." << status());
+    debugs(93,7, "initialized." << status());
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.